ts/src/program/namespace/account.ts

```diff
@@ -152,6 +153,34 @@ export class AccountClient<T = any> {
     return data;
   }
 
+  /**
+   * Returns multiple deserialized accounts.
+   * Accounts not found or with wrong discriminator are returned as null.
+   *
+   * @param addresses The addresses of the accounts to fetch.
+   */
+  async fetchMultiple(addresses: Address[]): Promise<(Object | null)[]> {
+    const accounts = await rpcUtil.getMultipleAccounts(
+      this._provider.connection,
+      addresses.map((address) => translateAddress(address))
+    );
+
+    const discriminator = await accountDiscriminator(this._idlAccount.name);
+    // Decode accounts where discriminator is correct, null otherwise
```

Comment:
```suggestion
    // Decode accounts where discriminator is correct, null otherwise.
```
```suggestion
    // Decode accounts where discriminator is correct, null otherwise
```